app/controllers/friendships_controller.rb

```diff
@@ -24,13 +22,19 @@ def show; end
 
   def update
     @friendship = Friendship.find(params[:id])
-    redirect_to users_path if (@friendship.confirmed == false) && @friendship.accept_request
+    @friendship.confirmed = true
+
+    if @friendship.save
+      @friendship.confirm_friend
+      redirect_to user_path(current_user.id), notice: 'Friend request was successfully confirmed.'
+    else
+      redirect_to user_path(current_user.id), alert: @friendship.error.full_messages.join('. ').to_s
+    end
   end
 
   def destroy
```

Comment:
- When i try to decline a friend request i have this error:
![Screen Shot 2021-09-20 at 14 12 19](https://user-images.githubusercontent.com/22258274/134061392-d2231c1c-eff1-427a-a62c-b74fd0db5dfc.png)